src/updateApis.ts

```diff
@@ -8,15 +8,7 @@
 import path from "path";
 import fs from "fs-extra";
 import { updateApis } from "./lib/utils";
-
-const API_FAMILIES = [
-  "pricing",
-  "customer",
-  "checkout",
-  "search",
-  "product",
-  "cdn",
-];
+import { API_FAMILIES } from "./lib/config";
```

Comment:
Should the other constants in this file be moved too?

Reply:
fine 🙄